src/pages/Profile.tsx

```diff
@@ -32,10 +33,21 @@ function Profile() {
     fetchPolicy: "network-only"
   });
   const needs = needsData?.getAllNeeds?.needs;
+  const { data: leadsData } = useQuery(GET_LEADS, {
+    variables: {
+      request: {
+        filters: {
+          authorId: cookies[USER_ID_COOKIE_NAME]
```

Comment:
Since we now use userId three times, it would be better to extract it to the top (const userId = cookies[USER_ID_COOKIE_NAME])

Reply:
Ok